R/expect-named.R

```diff
@@ -39,21 +39,48 @@ expect_named <- function(
     if (identical(act$names, NULL)) {
       msg <- sprintf("%s does not have names.", act$lab)
       return(fail(msg))
+    } else {
+      return(pass(act$val))
```

Comment:
The vignette doesn't explicitly mention early return `pass()` but it felt natural here

Reply:
Yeah, that looks reasonable to me